src/common/cscs_api_common.py

```diff
@@ -20,7 +20,21 @@
 import io
 import time
 
-debug = os.environ.get("F7T_DEBUG_MODE", None)
+# Checks if an environment variable injected to F7T is a valid True value
+# var <- str
```

Comment:
I am sorry, I know it's nitpicking, but this line is not actually true. You don't need the argument to be a string, and we even call it with a non string. I think this is better:
```suggestion
# var <- object
```
but you can also just leave it as it is.

Reply:
You're correct. I will change it.